testing/postgresql-cron/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: G.J.R. Timmer <gjr.timmer@gmail.com>
+# Maintainer:
+pkgname=postgresql-cron
+pkgver=1.0.0
+pkgrel=0
+pkgdesc="CRON based scheduler fro postgresql 9.5+"
+url="https://github.com/citusdata/pg_cron"
+arch="all"
+license="custom"
+depends="postgresql"
+makedepends="postgresql-dev"
+install="postgresql-cron.post-install"
+pkgusers="postgres"
+pkggroups="postgres"
+subpackages=""
+source="https://github.com/citusdata/pg_cron/archive/v1.0.0-rc.1.tar.gz"
```

Comment:
Source files must have unique name across all aports. Also do not hard-code version number, use variable, to make bumping version easier.

```diff
- source="https://github.com/citusdata/pg_cron/archive/v1.0.0-rc.1.tar.gz"
+ source="$pkgname-$pkgver.tar.gz::https://github.com/citusdata/pg_cron/archive/v$_pkgver.tar.gz"
```